### ISSUE TITLE
Tables don't dump in a consistent order

### ISSUE THREAD

**Issue by tovodeverett** [yamldb/yaml_db#50]
I'm having issues comparing dump files generated by `yaml_db` because the tables don't dump in a consistent order.  Records within a table dump in a consistent order because they are sorted by the first column, but the tables themselves can be returned in a random order from some database adapters.  I resolved this with a simple monkey-patch, but before I work up a pull request I'd like to see if there's a consensus that resolving this is desirable.

Here's my monkey-patch (all it does is add a sort to the end of the chain):
```
module SerializationHelper
  class Dump
    def self.tables
      ActiveRecord::Base.connection.tables.reject { |table| ['schema_info', 'schema_migrations'].include?(table) }.sort
    end
  end
end
```

**Comment by driehuis** [comment]
Ran into the same issue, in my case with sqlite3 (and probably older drivers, I'm trying to migrate an older app to Rails 4.2 and I'm diffing the database dump as an additional sanity check). I'll submit a pull request.

**Comment by dnrce** [comment]
Fixed by #80.
